tasks/src/test/java/com/redhat/parodos/tasks/TibcoWorkFlowTaskTest.java

```diff
@@ -0,0 +1,73 @@
+// @formatter:off
+package com.redhat.parodos.tasks;
+
+import com.redhat.parodos.workflow.context.WorkContextDelegate;
+import com.redhat.parodos.workflow.exception.MissingParameterException;
+import com.redhat.parodos.workflows.work.WorkContext;
+import com.redhat.parodos.workflows.work.WorkReport;
+import com.redhat.parodos.workflows.work.WorkStatus;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import javax.jms.JMSException;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+public class TibcoWorkFlowTaskTest {
+
+	String caFile = "/cafile";
+	String url = "ssl://localhost:7222";
+	String username = "username";
+	String passowrd = "password";
+	String topic = "topic";
+	String message = "test message";
```

Comment:
I'd suggest to make this `private final`

Reply:
why?

Reply:
`private` = it's a best practice to keep the scope as narrow as you can.
`final` =  It's a good practice to declare final a variable to be sure that it won't be modified unintentionally, and in addition it makes the understanding of code easier.

TBH this strings can be declared `private static final` so we're stating that such variables are constants and that they are associated with the class (not the instance of the class)

Reply:
@gciavarrini fixed